From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryder Belserion <no-reply@ryderbelserion.com>
Date: Thu, 29 Feb 2024 11:29:16 -0500
Subject: [PATCH] Update buildscript


diff --git a/build.gradle.kts b/build.gradle.kts
index 230451f467b79d081048bffbe09903c71f6d4896..04ad86feb331036709b50ab18ba25b6a533531a2 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -1,10 +1,10 @@
 import io.papermc.paperweight.util.*
 
+// Kelp start
 plugins {
-    java
-    `maven-publish`
-    id("com.github.johnrengelman.shadow")
+    id("java-plugin")
 }
+// Kelp end
 
 val log4jPlugins = sourceSets.create("log4jPlugins")
 configurations.named(log4jPlugins.compileClasspathConfigurationName) {
@@ -13,7 +13,14 @@ configurations.named(log4jPlugins.compileClasspathConfigurationName) {
 val alsoShade: Configuration by configurations.creating
 
 dependencies {
+    // Kelp start
+    implementation(libs.asm.commons)
+
+    implementation(libs.snakeyaml)
+    // Kelp stop
+
     implementation(project(":kelp-api")) // Purpur // Kelp
+
     implementation("io.papermc.paper:paper-mojangapi:${project.version}") {
         exclude("io.papermc.paper", "paper-api")
     }
@@ -35,7 +42,6 @@ dependencies {
     implementation("io.netty:netty-codec-haproxy:4.1.97.Final") // Paper - Add support for proxy protocol
     // Paper end
     implementation("org.apache.logging.log4j:log4j-iostreams:2.19.0") // Paper - remove exclusion
-    implementation("org.ow2.asm:asm-commons:9.5")
     implementation("org.spongepowered:configurate-yaml:4.2.0-SNAPSHOT") // Paper - config files
     implementation("commons-lang:commons-lang:2.6")
     implementation("net.fabricmc:mapping-io:0.5.0") // Paper - needed to read mappings for stacktrace deobfuscation
@@ -53,7 +59,6 @@ dependencies {
     runtimeOnly("org.apache.maven.resolver:maven-resolver-transport-http:1.9.18")
 
     // Pufferfish start
-    implementation("org.yaml:snakeyaml:1.32")
     implementation ("com.github.carleslc.Simple-YAML:Simple-Yaml:1.8.4") {
         exclude(group="org.yaml", module="snakeyaml")
     }
